MerlinAU.sh

```diff
@@ -6245,11 +6270,12 @@ Please manually update to version ${GRNct}${MinSupportedFirmwareVers}${NOct} or
         # Stop Entware services WITHOUT exceptions BEFORE the F/W flash #
         _EntwareServicesHandler_ stop -noskip
 
-        ##-------------------------------------##
-        ## Added by Martinski W. [2024-Sep-15] ##
-        ##-------------------------------------##
+        ##------------------------------------------##
+        ## Modified by ExtremeFiretop [2024-Nov-15] ##
+        ##------------------------------------------##
         # Remove cron jobs from 3rd-party Add-Ons #
         _RemoveCronJobsFromAddOns_
+        _DelCronJobRunScriptHook_
```

Comment:
TO BE REMOVED?
(If and When) you come up with a better solution.

Reply:
> TO BE REMOVED? (If and When) you come up with a better solution.
>
It's no longer needed once my latest changes in the "`_AcquireLock_`" function function are merged.

Reply:
> Will test and advise of your change shortly! :)
>
Take your time. We have the rest of the weekend to properly test & validate that the initial issues during post-reboot have been addressed.

P.S. My wife is calling... Hold my beer!! :>)

Reply:
Done!